scripts/src/main/resources/scripts/command/gcviewer

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup start help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+GCVIEWER_HOME="${DEVON_IDE_HOME}/software/gcviewer"
+
+# gcviewer installation
+function doSetup(){
+
+  if [ "${1}" != "silent" ]
+  then
+    if [ -z "${GCVIEWER_VERSION}" ]
+    then
+      doEcho "Getting the latest release of GCViewer..."
+      GCVIEWER_VERSION=$(curl "https://github.com/chewiebug/GCViewer/tags" | awk -F "/tag/" '/GCViewer/ {print $2}' | sort -r | head -1 | awk -F "\">" '{print $1}')
+    fi 
+    do_not_extract="Not Empty"
+    doInstall "-" "${GCVIEWER_HOME}" "gcviewer" "${GCVIEWER_VERSION}" "" "" "" "" "" "" "${do_not_extract}"
+    doCreateGCViewerScript
+  fi
+  
+}
+
+function doStart(){
+
+  if [ ! -d "${GCVIEWER_HOME}" ]
+  then
+    doSetup
+  else
```

Comment:
See other commandlets. Your code is not starting `gcviewer` if it is not already installed. That is confusing and different to other commandlets. I will quickly fix it so we can merge this.

Reply:
Done.